Updating bestiary entries with omitted creatures

sparkcity opened the pull request:
Updates:
- Added Adjutant, Collector Assassin, Space Hound, Horned Space Hound, and Albino Space Hound
- Added icons Andromeda to experiment with usage

sparkcity committed:
Updating bestiary entries with omitted creatures

Updates:
- Added Adjutant, Collector Assassin, Space Hound, Horned Space Hound, and Albino Space Hound
- Added icons Andromeda to experiment with usage

bushongj commented:
Taking a look

bushongj commented:
@sparkcity Only issue is Space hound needs Warp and Throw removed from it's spellbook.

sparkcity committed:
bestiary update - correction and collectors

update:
- correction to space hound to make him less cool than his space hound brothers
- collector entries

bushongj committed:
Update bestiary-me5e.db

bushongj commented:
Added Skill proficiencies for collector general to branch. Everything else looks good!
